dwitter/static/js/feed.js

```diff
@@ -1,4 +1,5 @@
-var onDweetChanged = function() {
+function onDweetChanged() {
+  // eslint-disable-next-line newline-per-chained-call
   var charCount = $(this).parent().parent().parent().find('.character-count')[0];
```

Comment:
The Airbnb ESLint config specifies that function call chains with more than 3 calls must be broken into multiple lines. I don't think that's necessary here.